### DISCUSSION TITLE
CLI11_PARSE not working like suggested in the README

### DISCUSSION THREAD

**Discussion by LostInCompilation** [CLIUtils/CLI11#852] [Q&A] [answered]
In the README under the section Unicode is written, that I can simply use `CLI11_PARSE(app)` or `CLI11_PARSE(app, CLI::argc(), CLI::argv());`. However both ways do not work. The first gives the error `too few arguments`, the second cannot find `CLI::argc()` or `CLI::argv()`.
I'm using the single header version (CLI11.hpp) from the latest release. Am I missing something on how to use the macro like suggested in the README?

**Comment by henryiii** [comment, marked as answer]
It’s not in a released version yet.
